script/apps/cassandra/deprovision/cassandra-app-deprovision-jiva

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -x
+
+#################
+## ENVIRONMENT ##
+#################
+
+## https://github.com/openebs/litmus/blob/master/apps/cassandra/deployers/test_vars.yml
+test_name=$(${utils_path}/generate_test_name testcase=cassandra-deployment metadata="")
```

Comment:
cassandra-deprovision as test_name?

Reply:
it seems all the test_name need to be changed

Reply:
As we are using same litmus jobs for both app provisoning and de-provisioning same name is used @vishnuitta . The litmus result is updated using label. For ex: label='app:cassandra-deprovision-litmus'

Reply:
Lets make use of the `run_id ` (i.e., metadata arg) to differentiate the action.

Reply:
ok @ksatchit, it make sense to use run_id to differentiate the provision and deprovisoning actions. I will make these changes and push. Thanks

Reply:
Done @ksatchit Can you please relook if the changes are good?